de.json

```diff
@@ -940,7 +940,7 @@
 			"option-show-attachments-description": "Zeige Anhänge, die in den Notizen enthalten sind.",
 			"option-show-existing-files-only": "Nur existierende Dateien zeigen",
 			"option-show-existing-files-only-description": "Wenn aktiviert, werden Links zu nicht-existierenden Dateien nicht angezeigt.",
-			"option-show-orphans": "Verwaistes Element",
+			"option-show-orphans": "Verwaiste Dateien",
```

Comment:
Ich finde das "Element" hier besser gepasst hat.

Reply:
Im englischen Original sind es 'files that are not linked to other files', deshalb der Wechsel zu 'Dateien' - ich würde hier auch nicht 'Elemente' verwenden, da das nirgends sonst in Obsidian ein Begriff ist.